var/config/extensions.php

```diff
@@ -7,6 +7,8 @@
         "OutputDataConfigToolkitBundle\\OutputDataConfigToolkitBundle" => TRUE,
         "Web2PrintToolsBundle\\Web2PrintToolsBundle" => TRUE,
         "Pimcore\\Bundle\\DataHubBundle\\PimcoreDataHubBundle" => TRUE,
-        "Pimcore\\Bundle\\ObjectMergerBundle\\ObjectMergerBundle" => TRUE
+        "Pimcore\\Bundle\\ObjectMergerBundle\\ObjectMergerBundle" => TRUE,
+        "Pimcore\\Bundle\\EcommerceFrameworkBundle\\PimcorePaymentProviderUnzerBundle" => TRUE,
+        "Pimcore\\Bundle\\EcommerceFrameworkBundle\\PimcorePaymentProviderDatatransBundle" => TRUE
```

Comment:
since datatrans is not implemented, let's remove it completely

Reply:
removed